drivers/ieee802154/ieee802154_mcr20a.h

```diff
@@ -25,9 +25,7 @@ struct mcr20a_context {
 	struct gpio_callback irqb_cb;
 	struct device *spi;
 	struct spi_config spi_cfg;
-#if defined(CONFIG_IEEE802154_MCR20A_GPIO_SPI_CS)
```

Comment:
Make it build if only DT_NXP_MCR20A_0_CS_GPIO_CONTROLLER is present, no?

Reply:
I was torn on this, I removed it if we ever support multiple instances of the driver.

Reply:
keep the ifdef, let's handle multiple instances another time. (when we'll have proper device generation DT based)

Reply:
Done.